### ISSUE TITLE
[NPC]  Vhulgra <Wind Rider Master> -12616- Ashenvale

### ISSUE THREAD

**Issue by Echinaceax** [WoWManiaUK/Blackwing-Lair#942]
**Links:**

https://www.wowhead.com/npc=12616/vhulgra

![splintertree fp](https://user-images.githubusercontent.com/39439201/54495431-df248780-48e3-11e9-907b-2775dce345bf.jpg)

_I have all the FP in Ashenvale_

![i do fp](https://user-images.githubusercontent.com/39439201/54495450-15fa9d80-48e4-11e9-9547-ebfbf446c176.jpg)

If i manually fly myself too Silverwind refuse, i can go back too splintertree post

**What is happening:**
- Cant use Splintertree post to other FP in Ashenvale or anywhere else
- I can from other Ashenvale FP just not splintertree post

![from silverwind refuge](https://user-images.githubusercontent.com/39439201/54495583-28290b80-48e5-11e9-90d3-18db6b472f9b.jpg)

**What should happen:**
- Should be able to use splintertree FP to other zones also

**Comment by golosomie** [comment]
Can confirm on PTR. Here is a list of flight points that should be available from Splintertree post:

- Valormok, Azshara
- Bilgewater Harbor, Azshara
- Orgrimmar
- The Crossroads, Northern Barrens
- The Mor'Shan Ramparts, Ashenvale
- Silverwind Refuge, Ashenvale
- Hellscream's Watch, Ashenvale
- Zoram'gar Outpost, Ashenvale
- Emerald Sanctuary, Felwood

**Comment by Echinaceax** [comment]
full flight paths should be checked and connecting properly @mfranca0009

**Comment by golosomie** [comment]
these are the flightpaths missing for the flight master of Zoram'gar Outpost ([Andruk](https://www.wowhead.com/npc=11901/andruk)):

- Hellscream's Watch, Ashenvale
- Splintertree Post, Ashenvale
- The Crossroads, Northern Barrens
- Cliffwalker Post, Stonetalon Mountains
- Sun Rock Retreat, Stonetalon Mountains
- Thunder Bluff
- Emerald Sanctuary, Felwood

**Comment by Echinaceax** [comment]
update:
Tested on new core - still bugged

I think i saw why

Splintertree Post npc https://www.wowhead.com/npc=12616/vhulgra had the green (!) 
Zoram'gar Outpost https://www.wowhead.com/npc=11901/andruk had the green (!) 

Thraka <Wind Rider Master>https://cata-twinhead.twinstar.cz/?npc=34429 and Wind Tamer Shoshok <Flight Master>https://cata-twinhead.twinstar.cz/?npc=34943  dont 

i think some of the "discovery (!) markers are missing @mfranca0009 so i would fixed these 2 and retest

**Comment by mfranca0009** [comment]
@Echinaceax 
Completely unrelated, it's a client issue. New core will do nothing in regards to fixing this. The discovery marker over their head has nothing to do with it....

It needs proper flight paths set in DBC, that's the proper way of fixing it. Other than that, it could be hackfixed by scripting and spawning the flight NPC and forcing the player to mount it, then proceeding to start a smooth waypoint path to the place they wanted to go.

Better off to fix this the correct way, and avoid the hackfix.

**Comment by Echinaceax** [comment]
@mfranca0009 
Yes i wasnt sure tbh, I added my thoughts as i retested new core. as you know from the list :)

ofc, we have to fix it the correct way :)
